pydantic_ai_slim/pydantic_ai/profiles/openai.py

```diff
@@ -71,12 +71,16 @@ def openai_model_profile(model_name: str) -> ModelProfile:
     # See https://github.com/pydantic/pydantic-ai/issues/974 for more details.
     openai_system_prompt_role = 'user' if model_name.startswith('o1-mini') else None
 
+    # Enable response prefix for DeepSeek and OpenRouter models
+    supports_response_prefix = 'deepseek' in model_name.lower() or 'openrouter' in model_name.lower()
```

Comment:
We should add this to `deepseek_model_profile` and `OpenRouterProvider.model_profile` instead